src/test/java/de/mediathekview/mserver/crawler/zdf/tasks/ZdfIndexPageTaskTest.java

```diff
@@ -0,0 +1,70 @@
+package de.mediathekview.mserver.crawler.zdf.tasks;
+
+import static org.hamcrest.CoreMatchers.notNullValue;
+import static org.hamcrest.MatcherAssert.assertThat;
+import static org.hamcrest.Matchers.equalTo;
+
+import de.mediathekview.mserver.crawler.zdf.ZdfConfiguration;
+import de.mediathekview.mserver.crawler.zdf.ZdfConstants;
+import de.mediathekview.mserver.testhelper.JsoupMock;
+import java.util.Arrays;
+import java.util.Collection;
+import java.util.Optional;
+import org.jsoup.Jsoup;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.junit.runners.Parameterized;
+import org.powermock.core.classloader.annotations.PowerMockIgnore;
+import org.powermock.core.classloader.annotations.PrepareForTest;
+import org.powermock.modules.junit4.PowerMockRunner;
+import org.powermock.modules.junit4.PowerMockRunnerDelegate;
+
+@RunWith(PowerMockRunner.class)
+@PrepareForTest({Jsoup.class})
+@PowerMockIgnore("javax.net.ssl.*")
+@PowerMockRunnerDelegate(Parameterized.class)
+public class ZdfIndexPageTaskTest {
+
+  @Parameterized.Parameters
+  public static Collection<Object[]> data() {
+    return Arrays.asList(new Object[][]{
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove this array creation and simply pass the elements. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS3878)